app/src/main/java/com/sereno/serenoandroidapp/services/OpenWeatherMapService.kt

```diff
@@ -0,0 +1,16 @@
+package com.sereno.serenoandroidapp.services
+
+import com.sereno.serenoandroidapp.models.apis.openweathermap.currentweatherdata.CurrentWeatherResponce
+import retrofit2.Call
+import retrofit2.http.GET
+import retrofit2.http.Query
+
+interface OpenWeatherMapService {
+    @GET("data/2.5/weather/")
+    fun fetchWeather(
```

Comment:
```suggestion
    fun getCurrentWeatherData(
```

getCurrentWeatherDataはこっちのことです

Reply:
app/src/main/java/com/sereno/serenoandroidapp/activities/WeatherInfoActivity.ktの中のfunの名前はどのようにしたらよいですか？